homework/01LessonOne.js

```diff
@@ -203,12 +210,12 @@ var stringFourStrictNotEqualsNumberFour;
 // What’s the difference between == and ===?
 //
 // Your Answer Goes Here:
-//
+//The "==" operator means "equal to". The "===" operator means equal to in value and equal to in type.
 //
 // What’s the difference between != and !==?
 //
 // Your Answer Goes Here:
-//
+//The "!=" operator means not equal. The "!==" operator means not equal in value or not equal in type.
```

Comment:
Good job. **Strict equality** is the key phrase here.

[MDN Equality comparisons and sameness](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness)